CODEOWNERS

```diff
@@ -0,0 +1,80 @@
+/cpu/cc26xx_cc13xx/                         @hexluthor @smlng
+/cpu/cortexm_common/                        @haukepetersen @thomaseichinger @DipSwitch
+/cpu/efm32/                                 @basilfx
+/cpu/esp*/                                  @gschorcht
+/cpu/kinetis/                               @gebart
+/cpu/lpc2387/                               @benpicco @maribu
+/cpu/mips*/                                 @kaspar030 @francois-berder
+/cpu/msp430*/                               @OlegHahm
+/cpu/sam0_common/                           @benpicco @dylad @keestux
+/cpu/samd21/                                @dylad @biboc
+/cpu/saml1x/                                @dylad
+/cpu/saml21/                                @dylad
+/cpu/stm32*/                                @aabadie @fjmolinas @DipSwitch @vincent-d
+
+/drivers/at86rf2xx/                         @daniel-k @gebart @Hyungsin @jia200x @smlng @miri64
+/drivers/cc110x/                            @maribu
+/drivers/cc1xxx_common/                     @maribu
+/drivers/ccs811/                            @gschorcht
+/drivers/enc28j60/                          @haukepetersen
+/drivers/dcf77/                             @daexel
+/drivers/dose/                              @jue89
+/drivers/itg320x/                           @gschorcht
+/drivers/mrf24j40/                          @bergzand
+/drivers/pca9685/                           @gschorcht
+/drivers/sht3x/                             @gschorcht
+/drivers/si70xx/                            @basilfx
+/drivers/sx127x/                            @aabadie @jia200x
+/drivers/ws281x/                            @maribu
+/drivers/xbee/                              @haukepetersen @miri64
+
+/pkg/cryptoauthlib/                         @Einhornhool @PeterKietzmann
+/pkg/lua/                                   @jcarrano
+/pkg/lwip/                                  @miri64
+/pkg/gecko_sdk/                             @basilfx
+/pkg/micropython/                           @bergzand
+/pkg/openthread/                            @Hyungsin @jia200x
+/pkg/u8g2/                                  @basilfx
+/pkg/ucglib/                                @basilfx
+
+/sys/net/                                   @miri64 @haukepetersen @PeterKietzmann @cgundogan
+/sys/usb/                                   @bergzand @dylad @aabadie
+/sys/xtimer/                                @kaspar030 @gebart @MichelRottleuthner
+/sys/net/application_layer/gcoap/           @haukepetersen @kb2ma
+/sys/net/application_layer/nanocoap/        @kaspar030 @haukepetersen
+/sys/net/gnrc/routing/rpl/                  @cgundogan @emmanuelsearch
+/sys/random/fortuna/                        @basilfx
+
+/tests/                                     @miri64 @MrKevinWeiss @smlng @leandrolanzieri @aabadie @MichelRottleuthner
+/tests/cpu_efm32_features/                  @basilfx
+
+Kconfig                                     @leandrolanzieri
+*.md                                        @aabadie @jia200x
```

Comment:
Does that mean all `*.md` files in the repo, no matter in which directory? Isn't this a little bit vague?

Reply:
That is also the case for `sys/net` ;-). It's just to point GitHub in the right direction about who to assign. We still can make things more precise later on.

Reply:
I guess that even if the "wrong" maintainer is occasionally flaged, this will still serve the intended goal: That maintainer is likely aware who would be the right person to dial in and juat do so.

Reply:
That's for @aabadie and @jia200x to decide (though, I don't doubt that they would happily be triggered for each md file (: )

Reply:
I would prefer to limit this rules on md files present in the doc directory. Otherwise, I will be triggered for each changes in application README, even the one I don't want to review.

Reply:
Does anyone know if only the first match is triggered (so that @aabadie  and @jia200x would only get triggered for .md files that don't fall into other categories), or all matches?

Reply:
From the docs:
```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2

# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner
```

>Order is important; the last matching pattern takes the most precedence.

Reply:
> Order is important; the last matching pattern takes the most precedence.

right, so we should put these to the start of the file.

Reply:
Actually `Kconfig` should still be at the end, so our KConfig maintainer is always in the loop about changes to KConfig since those require some global coherence.